sql/understanding-the-query-execution-plan.md

```diff
@@ -14,7 +14,7 @@ The result of the `EXPLAIN` statement provides information about how TiDB execut
 
 - `EXPLAIN` works together with `SELECT`, `DELETE`, `INSERT`, `REPLACE`, and `UPDATE`.
 - When you run the `EXPLAIN` statement, TiDB returns the final physical execution plan which is optimized by the SQL statement of `EXPLAIN`. In other words, `EXPLAIN` displays the complete information about how TiDB executes the SQL statement, such as in which order, how tables are joined, and what the expression tree looks like. For more information, see [`EXPLAIN` output format](#explain-output-format).
-- TiDB does not support `EXPLAIN [options] FOR CONNECTION connection_id` currently. We'll do it in the future. For more information, see [#4351](https://github.com/pingcap/tidb/issues/4351).
+- TiDB now supports `EXPLAIN [options] FOR CONNECTION connection_id` which works differently than `EXPLAIN FOR` in MySQL. See [`EXPLAIN FOR CONNECTION`](#explain-for-connection)。
```

Comment:
```suggestion
- TiDB also supports `EXPLAIN [options] FOR CONNECTION connection_id`, but with minor differences from MySQL. See [`EXPLAIN FOR CONNECTION`](#explain-for-connection).
```

- I don't think users need the history that TiDB did not support EXPLAIN FOR CONNECTION.
- I think the semantic differences are only minor.  We should not scare users it is completely different, given that we have already explained the output format of EXPLAIN is different.